chsdi/templates/htmlpopup/kgs_anhoerung.mako

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <% c['stable_id'] = True %>
+    <tr><td class="cell-left">${_('ch.babs.kulturgueter.zkob')}</td>   <td>${c['attributes']['zkob']}</td></tr>
+    <tr><td class="cell-left">${_('y')}</td>              <td>${int(round(c['attributes']['x'],0)) or '-'}</td></tr>
+    <tr><td class="cell-left">${_('x')}</td>              <td>${int(round(c['attributes']['y'],0)) or '-'}</td></tr>
+    <tr><td class="cell-left">${_('gemeinde')}</td>       <td>${c['attributes']['gemeinde'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('kanton')}</td>         <td>${c['attributes']['kt_kz'] or '-'}</td></tr>
+</%def>
+
+<%def name="extended_info(c, lang)">
+    <%
+        c['stable_id'] = True
+        objarts = c['attributes']['objektart'].split(',')
+        import csv
+        from urllib2 import urlopen
+        dataGeoAdminHost = request.registry.settings['datageoadminhost']
+        dataPath = 'ch.babs.kulturgueter'
+        csv_url = "https://" + dataGeoAdminHost + "/ch.babs.kulturgueter-anhoerung/image/meta.txt"
```

Comment:
is it really planned to use the same meta.txt as for the official layer?
https://data.geo.admin.ch/ch.babs.kulturgueter-anhoerung/image/meta.txt

Reply:
No its a new one.

Reply:
ok.